content/10-minute-tutorial.md

```diff
@@ -1,3 +1,328 @@
 ---
 title: 10 Minute Tutorial
+polyglot: true
 ---
+{{% block "java" %}}
+
+To get started with Cucumber in Java, you will need the following:
+
+- [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html) - Java 8 (Java 9 is not yet supported by Cucumber)
+
+- [Maven](https://maven.apache.org/index.html) - Version 3.3.1 or higher
+
+- An IDE editor, for example [IntelliJ IDEA](https://www.jetbrains.com/idea/?fromMenu#chooseYourEdition) (which will be used in this
+  introduction)
+
+- A Cucumber plugin for your IDE, for example [IntelliJ IDEA Cucumber for Java plugin](https://plugins.jetbrains.com/plugin/7212-cucumber-for-java) to go with IntelliJ IDEA
+
+# Setting up the project
+
+First, we need to set up the project so we can use Cucumber.
+
+## Create a Maven project
+
+To create a new Maven project in IntelliJ:
+
+1. Click the menu option **File > New > Project**
+
+2. In the **New project** dialog box, select **Maven** on the left (if it isn't already selected)
+
+3. Make sure that the Project SDK is selected (for instance, Java 1.8) and click **Next**
+
+4. Specify a **GroupId** and **ArtifactId** for your project and click **Next**
```

Comment:
Should we tell people what these are for? Or do we assume they already know because they are setting up a Java project?

Reply:
Ad 3. Unfortunately cannot test what happens *exactly* as I have Java set up, so cannot describe in detail here. But I've helped coworkers who did not... and you run into interesting problems (stuff not getting recognized mainly)
Ad 4. If you don't, you cannot create the project.

For this tutorial, I'm going to assume they know why OR will just follow the steps (I only mention 3 because I've seen this lead to problems).